zz/zzcd.sh

```diff
@@ -0,0 +1,141 @@
+# ----------------------------------------------------------------------------
+# Mudança de diretório com histórico de mudanças acessível por menu.
+# Funcionamento idêntico à função embutida cd, com as seguintes diferenças:
+#
+# 1. Quando chamada sem opção ou parâmetros, exibe o histórico de diretórios,
+# com números correspondentes que podem ser usados na navegação
+# 2. Quando chamada com a opção -i sem parâmetros, exibe um prompt interativo
+# a partir do histórico de diretórios
+# 3. Quando chamada com um parâmetro numérico, muda o diretório atual,
+# com histórico, para o índice correspondente listado no item 1. acima
+#
+# Uso: zzcd [ -i | [diretório ou índice do diretório no histórico] ]
+# Ex.: zzcd /usr/local/bin, ou zzcd [ índice ], ou zzcd -i
+#
+# Autor: Nilo César Teixeira (nilo.teixeira@gmail.com)
+# Desde: 2021-07-07
+# Versão: 1
+# Requisitos: dirs pushd
+# Tags: cd, mudança de diretórios, histórico, menu de diretórios
+# ----------------------------------------------------------------------------
+zzcd ()
+{
+
+	# Função auxiliar que muda para um diretório exibido em $(dirs -v),
+	# a partir de um parâmetro de pesquisa
+	zzcd_auxiliar ()
+	{
+		local dirs numdir
+
+		numdir="$1"
+		dirs="$2"
+		# Display de $(dirs -v) tem um espaço inicial
+		# e dois espaços antes do diretório
+		numdir="$(echo "$dirs" | grep -E "^ $numdir  " | cut -d ' ' -f 2)"
+		# Omitindo erro de pilha vazia
+		pushd "+$numdir" 2>&1 >/dev/null
```

Comment:
```suggestion
		pushd "+$numdir" >/dev/null 2>&1 
```

Aqui a ordem importa. Primeiro você silencia a saída padrão redirecionando ela pra `/dev/null` e só depois informa que a saída de erro deve usar o mesmo destino que a saída padrão está atualmente usando.